pkg/utils/functional/functional.go

```diff
@@ -31,6 +31,16 @@ func UnionStringMaps(maps ...map[string]string) map[string]string {
 	return result
 }
 
+func StringsWithout(vals []string, remove string) []string {
```

Comment:
Maybe call this StringSliceWithout to match the next fxn name in the file?

Reply:
Also it's a little surprising this doesn't return a (modified) copy of the string slice rather than the original.

Reply:
I changed it so that it creates a new slice, so that it won't mess with the original slice.